lua/oil/config.lua

```diff
@@ -2,7 +2,7 @@ local default_config = {
   -- Id is automatically added at the beginning, and name at the end
   -- See :help oil-columns
   columns = {
-    "icon",
+    { "icon", default_file = "", directory = " " },
```

Comment:
Since this default config doubles as documentation and these options are more for advanced config, I'd prefer to leave it as "icon" here. The documentation of the new config options can go in https://github.com/stevearc/oil.nvim/blob/14a914aef1db4bf4b43d98d1bcf29754ea7997fc/.github/generate.py#L103

See the lines above for an example of how to use LuaParam. It should be a straightforward change, but if you need to test it locally you can run `python .github/main.py generate`

Reply:
I don't understand what you are talking about...